sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -399,6 +399,11 @@ def translate_output_properties(output: Any,
     # Unwrap optional types.
     typ = _types.unwrap_optional_type(typ) if typ else typ
 
+    # If the typ is Any, set it to None to treat it as if we don't have any type information,
+    # to avoid raising errors about unexpected types, since it could be any type.
+    if typ is Any:
+        typ = None
```

Comment:
wow python is weird. so it expects a match? only `Any == Any`? `List` does not qualify as `Any`?

Reply:
Not a Python oddity, we're raising the error:

https://github.com/pulumi/pulumi/blob/910aa96016cd325f4ad5b808ef5075ad68699544/sdk/python/lib/pulumi/runtime/rpc.py#L440-L451

We just weren't taking into account that `typ` may be `Any`, in which case, we shouldn't raise the error.

Reply:
ah got it thank you!